migrations/build.gradle.kts

```diff
@@ -0,0 +1,25 @@
+buildscript {
+    dependencies {
+        classpath(libs.postgres)
+    }
+}
+
+plugins {
+    kotlin("jvm")
+    id("org.flywaydb.flyway")
+}
+
+java.sourceCompatibility = JavaVersion.VERSION_11
+java.targetCompatibility = JavaVersion.VERSION_11
+
+val postgresHost = System.getenv("DB_HOST") ?: "127.0.0.1"
+val postgresPort = System.getenv("DB_PORT") ?: "5432"
+
+flyway {
```

Comment:
Do we need to actually include the flyway plugin configs in this repo?  Maybe just having the files is enough.  They might even be able to live in the server's resource folder instead?

Reply:
that's a fair point, I guess if just running the app migrates stuff then we probably don't have a need to run it manually via gradle